lib/Controller/ChatController.php

```diff
@@ -462,6 +468,64 @@ public function receiveMessages(int $lookIntoFuture,
 		return $response;
 	}
 
+	/**
+	 * @NoAdminRequired
+	 * @RequireParticipant
+	 * @RequireReadWriteConversation
+	 * @RequireModeratorOrNoLobby
+	 *
+	 * @param int $messageId
+	 * @return DataResponse
+	 */
+	public function deleteMessage(int $messageId): DataResponse {
+		try {
+			$message = $this->chatManager->getComment($this->room, (string) $messageId);
+		} catch (NotFoundException $e) {
+			return new DataResponse([], Http::STATUS_NOT_FOUND);
+		}
+
+		$attendee = $this->participant->getAttendee();
+		if (!$this->participant->hasModeratorPermissions(false)
+			&& ($message->getActorType() !== $attendee->getActorType()
+				|| $message->getActorId() !== $attendee->getActorId())) {
+			// Actor is not a moderator or not the owner of the message
+			return new DataResponse([], Http::STATUS_FORBIDDEN);
+		}
+
+		$maxDeleteAge = $this->timeFactory->getDateTime();
+		$maxDeleteAge->sub(new \DateInterval('PT6H'));
```

Comment:
I guess not much use of making this configurable ? (through occ at least)

Reply:
No, we don't support deleting after 6h